application/include/mmul.hpp

```diff
@@ -24,16 +24,15 @@ using namespace executor;
 using namespace Eigen;
 
 template <typename Executor,
-          typename executor::instance_of_base<inline_executor, Executor> = 0>
+          typename executor::InstanceOf<Executor, inline_executor> = 0>
```

Comment:
Instead of `= 0` make it `= true` so it reads better

Reply:
I thought of that at some point, but even `= false` would be valid, so its slightly misleading.
`= 0` though slightly unclear is unambiguous and more common to see.